### ISSUE TITLE
Add cargo-release?

### ISSUE THREAD

**Issue by azerupi** [kbknapp/cargo-extras#23]
[sunng87/cargo-release](https://github.com/sunng87/cargo-release) seems like a useful cargo subcommand to add to the extras bundle.

**Comment by kbknapp** [comment]
Good idea! I need to go through and update each of these anyways so I'll add it in that pr.
